tests/test_sync.py

```diff
@@ -0,0 +1,73 @@
+import json
+from pytest import fixture
+import os
+
+from patientsearch.models import sync_bundle
+
+
+@fixture
+def external_patient_search(request):
+    data_dir, _ = os.path.splitext(request.module.__file__)
+    with open(os.path.join(
+            data_dir, "external_patient_search.json"), 'r') as json_file:
+        data = json.load(json_file)
+    return data
+
+
+@fixture
+def new_patient(request):
+    data_dir, _ = os.path.splitext(request.module.__file__)
+    with open(os.path.join(
+            data_dir, "new_patient_result.json"), 'r') as json_file:
+        data = json.load(json_file)
+    return data
+
+
+@fixture
+def internal_patient_miss(request):
+    data_dir, _ = os.path.splitext(request.module.__file__)
+    with open(os.path.join(
+            data_dir, "internal_patient_miss.json"), 'r') as json_file:
+        data = json.load(json_file)
+    return data
```

Comment:
could all the file-handling code be moved to one fixture, and each fixture based on that one would just pass the filename?

Reply:
how's the committed solution?

Reply:
Nice and DRY :) :100: